packages/ember-toucan-core/src/test-support/components/button.ts

```diff
@@ -2,7 +2,7 @@ import { click } from '@ember/test-helpers';
 
 import { PageObject } from 'fractal-page-object';
 
-export class Button extends PageObject {
+export class ButtonPageObject extends PageObject {
```

Comment:
Due to an export name change, should we consider a breaking-change changeset?  I'd be okay making it a `patch`, as I'm not sure anyone is using this directly.  As long as it's documented in the release notes 👍

Reply:
I'm cool with that. 

Though I tend to wait to add a changeset until after PR review in case there's some discussion. I'll make sure to add one after approval.

Reply:
Done.